genic/power.c

```diff
@@ -250,86 +274,8 @@ double sigma2_int(double k, void * params)
       w = 1./3. - kr2/30. +kr2*kr2/840.;
   else
       w = 3 * (sin(kr) / kr - cos(kr)) / kr2;
-  x = 4 * M_PI / (2 * M_PI * 2 * M_PI * 2 * M_PI) * k * k * w * w * PowerSpec(k);
+  x = 4 * M_PI / (2 * M_PI * 2 * M_PI * 2 * M_PI) * k * k * w * w * PowerSpec(k, 7);
```

Comment:
a type of -1 or perhaps using a bitmask might be cleaner than 7.

Reply:
Traditionally I have used 7 to mean "total power" in transfer function code, but I can make it -1 if you prefer.